src/pages/kiyoonkim/Login/Login.js

```diff
@@ -1,7 +1,62 @@
-import React from "react";
+import React, { useState } from "react";
+import "./Login.scss";
+import { useNavigate } from "react-router-dom";
 
-function Loginkimkiyoon() {
-  return <h1>this is login</h1>;
-}
+const Login = () => {
+  const navigate = useNavigate();
+  const [idValue, setIdValue] = useState("");
+  const [pwValue, setPwValue] = useState("");
 
-export default Loginkimkiyoon;
+  const saveUserId = function (event) {
+    setIdValue(idValue => {
+      idValue = event.target.value;
+      return idValue;
+    });
+  };
```

Comment:
- 최신 문법인 화살표 함수로 사용
- set 함수의 인자로 단순히 전달하면 됨
```suggestion
  const saveUserId = (e) => {
    setIdValue(e.target.value);
  };
```